docker/Dockerfile

```diff
@@ -50,6 +50,12 @@ RUN echo "## Aggressive optimization: Remove debug symbols" \
     && find ${EMSDK}/upstream/bin -type f -exec strip -s {} + || true \
     && echo "## Done"
 
+RUN echo "## Installing npm dependencies" \
```

Comment:
Maybe add a comment here, since npm dependencies are included by default in emsdk, just not the `devDependnecies`.

The emsdk build script uses does install npm dependencies, just no the `devDependencies`.  It uses `npm ci --production --no-optional`.

How about `emsdk install tsc` so that we its more obvious why we are doing this here?

Reply:
Hm, in that case perhaps emscripten should move `typescript` to `dependencies` instead.

Reply:
We decided we didn't want to add the tsc dependency for all users, and instead make it an optional dependencies that certain users can opt into.

We can possibly revisit that decision at a later date.